redbox-core/redbox/models/settings.py

```diff
@@ -118,14 +118,18 @@ class AISettings(BaseModel):
     condense_question_prompt: str = CONDENSE_QUESTION_PROMPT
     summarisation_system_prompt: str = SUMMARISATION_SYSTEM_PROMPT
     summarisation_question_prompt: str = SUMMARISATION_QUESTION_PROMPT
-    summarisation_chunk_max_tokens: int = 20_000
-    summarisation_max_concurrency: int = 128
+    map_max_concurrency: int = 128
     map_system_prompt: str = MAP_SYSTEM_PROMPT
     map_question_prompt: str = MAP_QUESTION_PROMPT
     map_document_prompt: str = MAP_DOCUMENT_PROMPT
     reduce_system_prompt: str = REDUCE_SYSTEM_PROMPT
     reduce_question_prompt: str = REDUCE_QUESTION_PROMPT
 
+    @computed_field  # type: ignore
+    @property
+    def stuff_chunk_max_tokens(self) -> int:
+        return int(self.context_window_size * 0.8)
```

Comment:
could we make the 0.8 another setting? stuff_chunk_context_ratio which defaults to 0.75